### ISSUE TITLE
syslog doesn't exist

### ISSUE THREAD

**Issue by obrie** [graalvm/truffleruby#149]
Looks like `syslog` doesn't exist:

```
ext-truffleruby :001 > require 'syslog'
LoadError: cannot load such file -- syslog
	from /home/aaron/Projects/truffle/truffleruby/lib/mri/rubygems/core_ext/kernel_require.rb:55:in `gem_original_require'
	from /home/aaron/Projects/truffle/truffleruby/lib/mri/rubygems/core_ext/kernel_require.rb:55:in `require'
	from (irb):1:in `<main>'
	from /home/aaron/.rvm/rubies/ext-truffleruby/bin/irb:15:in `<main>'
```

**Comment by chrisseaton** [comment]
`syslog` is implemented in C in MRI. We are starting to support C extensions in TruffleRuby but generally we'd rather use alternatives.

In Rubinius `syslog` is implemented using the FFI https://github.com/rubysl/rubysl-syslog/blob/2.0/lib/rubysl/syslog/syslog.rb.ffi. We would need to implement more of FFI to use this, and this licence line may be a bother https://github.com/rubysl/rubysl-syslog/blob/2.0/lib/rubysl/syslog/syslog.rb.ffi#L2.

JRuby use the same code from Rubinius. I'll ask them what they think about the licence.

**Comment by chrisseaton** [comment]
Solution in https://github.com/graalvm/truffleruby/pull/347

**Comment by eregon** [comment]
Fixed by #347.
